builtin/logical/pki/backend_test.go

```diff
@@ -6691,121 +6691,125 @@ func TestProperAuthing(t *testing.T) {
+		"issuers/generate/intermediate/internal":   shouldBeAuthed,
+		"issuers/generate/intermediate/existing":   shouldBeAuthed,
+		"issuers/generate/intermediate/kms":        shouldBeAuthed,
+		"issuers/generate/root/exported":           shouldBeAuthed,
+		"issuers/generate/root/internal":           shouldBeAuthed,
+		"issuers/generate/root/existing":           shouldBeAuthed,
+		"issuers/generate/root/kms":                shouldBeAuthed,
+		"issuers/import/cert":                      shouldBeAuthed,
+		"issuers/import/bundle":                    shouldBeAuthed,
+		"key/default":                              shouldBeAuthed,
+		"keys":                                     shouldBeAuthed,
+		"keys/generate/internal":                   shouldBeAuthed,
+		"keys/generate/exported":                   shouldBeAuthed,
+		"keys/generate/kms":                        shouldBeAuthed,
+		"keys/import":                              shouldBeAuthed,
+		"ocsp":                                     shouldBeUnauthedWriteOnly,
+		"ocsp/dGVzdAo=":                            shouldBeUnauthedReadList,
+		"revoke":                                   shouldBeAuthed,
+		"revoke-with-key":                          shouldBeAuthed,
+		"roles/test":                               shouldBeAuthed,
+		"roles":                                    shouldBeAuthed,
+		"root":                                     shouldBeAuthed,
+		"root/generate/exported":                   shouldBeAuthed,
+		"root/generate/internal":                   shouldBeAuthed,
+		"root/generate/existing":                   shouldBeAuthed,
+		"root/generate/kms":                        shouldBeAuthed,
+		"root/replace":                             shouldBeAuthed,
+		"root/rotate/internal":                     shouldBeAuthed,
+		"root/rotate/exported":                     shouldBeAuthed,
+		"root/rotate/existing":                     shouldBeAuthed,
+		"root/rotate/kms":                          shouldBeAuthed,
+		"root/sign-intermediate":                   shouldBeAuthed,
+		"root/sign-self-issued":                    shouldBeAuthed,
+		"sign-verbatim":                            shouldBeAuthed,
+		"sign-verbatim/test":                       shouldBeAuthed,
+		"sign/test":                                shouldBeAuthed,
+		"tidy":                                     shouldBeAuthed,
+		"tidy-cancel":                              shouldBeAuthed,
+		"tidy-status":                              shouldBeAuthed,
+		"unified-crl":                              shouldBeUnauthedReadList,
+		"unified-crl/pem":                          shouldBeUnauthedReadList,
+		"unified-crl/delta":                        shouldBeUnauthedReadList,
+		"unified-crl/delta/pem":                    shouldBeUnauthedReadList,
+		"unified-ocsp":                             shouldBeUnauthedWriteOnly,
+		"unified-ocsp/dGVzdAo=":                    shouldBeUnauthedReadList,
+		"acme/directory":                           shouldBeUnauthedReadList,
```

Comment:
You might end up wanting to add the ACME paths here in the same manner fwiw.

Reply:
Sure, will adjust in a later PR

Reply:
Ah I see why now looking at the diff in GH. Updated